packages/apps/docs/src/scripts/importReadme.mjs

```diff
@@ -119,6 +119,11 @@ const recreateUrl = (pages, url, root) => {
         url = `${url}#${slug}`;
       }
 
+      // remove double slashes from internal url, if any
+      if (!url.includes('http')) {
```

Comment:
is this an issue we have?

Reply:
yes! since using the next.js Link component, a few urls with double slashes in their path popped up, causing a warning that breaks the build